scripts/r.in.srtm/r.in.srtm.py

```diff
@@ -230,7 +230,7 @@ def main():
         try:
             zf = zfile.ZipFile(zipfile)
             zf.extractall()
-        except:
+        except Exception:
```

Comment:
```python
except (FileNotFoundError, PermissionError, OSError, zipfile.BadZipFile):
```

1. FileNotFoundError: This exception is raised during the zfile.ZipFile(zipfile) call if the file specified by zipfile does not exist.  
2. PermissionError: This exception is raised during the zfile.ZipFile(zipfile) call if the program does not have the necessary permissions to read the file specified by zipfile. It can also be raised during the zf.extractall() call if the program does not have the necessary permissions to write the extracted files and directories.  
3. zipfile.BadZipFile: This exception is raised during the zfile.ZipFile(zipfile) call if the file specified by zipfile is not a zip file or it is a corrupted zip file.  
4. OSError: This is a more general exception that might be raised during the zfile.ZipFile(zipfile) or zf.extractall() calls for a variety of reasons, such as if there's a problem with the file system.